### PR TITLE
Update commonError for Transacting-3

### PR DESCRIPTION
**Pull request by benalleng**
Updates the commonError following the update to the logic to accept the scriptPubKeyBytes string
image

### CODE REVIEW

**Comment by ecurrencyhodler** [comment]
I really like this addition suggestion. 

It catches the flow where a player tries to do something similar to the challenge 1. 

Tagging @GBKS and @satsie for their thoughts.

**Comment by GBKS** [comment]
Yes, love it. That's exactly the type of custom responses we want.

File: i18n/locales/en.ts [hunk 1]
```diff
@@ -8,7 +8,7 @@ const translations = {
     coming_soon: 'Coming soon',
     start_chapter: 'Start chapter',
   },
-///ABOUT PAGE
+  ///ABOUT PAGE
```
Comment (afterburn) [inline i18n/locales/en.ts:11]: Not sure why we need this comment but we can remove it later

**Review by afterburn — APPROVED** [review]

File: i18n/locales/en.ts (hunk 1 above)
Comment (satsie) [reply i18n/locales/en.ts:11]: These were added in https://github.com/saving-satoshi/saving-satoshi/pull/302

Comment (ecurrencyhodler) [reply i18n/locales/en.ts:11]: They're just headings so contributors can more easily navigate the doc.

**Pull request merged**